tests/features/timeline.feature

```diff
@@ -11,12 +11,35 @@ Feature: Timeline paragraph.
     And I fill in "Title" with "Timeline paragraph test"
     And I press "Add Timeline"
     Then the following fields should be present "Label, Title, Content, Expand button"
-
-    When I press "Save"
+    When I fill in "Label" with "Label" in the "first" "Items" field element
```

Comment:
Let's use a different value, something that has a bit more works, like "Item 1 label" and so on.